snmp/datadog_checks/snmp/data/profiles/meraki-cloud-controller.yaml

```diff
@@ -0,0 +1,34 @@
+# Profile for Meraki Cloud devices
+#
+# You need the MIB compiled to get the data. You can run the following command to get it:
+# $ /opt/datadog-agent/embedded/bin/python /opt/datadog-agent/embedded/bin/mibdump.py  --destination-directory=/opt/datadog-agent/embedded/lib/python2.7/site-packages/pysnmp_mibs MERAKI-CLOUD-CONTROLLER-MIB.mib
```

Comment:
We should remove these lines, as the SNMP `README.md` already contains instructions on installing MIBs. :-) See https://github.com/DataDog/integrations-core/pull/5342.

Reply:
👍 Sounds good